lib/diagnostic.js

```diff
@@ -31,28 +41,36 @@ db.dropDatabase();
 // Daenerys Targaryen
 // Jon Arryn
 
-// your code begins here
+db.houses.update(
+   { name : "House Stark" },
+   { $push: { members: { $each: [ 'Ned Stark', 'Arya Stark' ] } } }
+);
 
-// your code ends here
+db.houses.update(
+   { name : "House Targaryen" },
+   { $push: { members: { $each: [ 'Viserys Targaryen', 'Daenerys Targaryen' ] } } }
+);
 
+db.houses.update(
+   { name : "House Arryn" },
+   { $push: { members: { $each: [ 'Jon Arryn' ] } } }
+);
 
 // Question 3
 //
 // House Arryn is not honorable!  Delete their motto.
 
-// your code begins here
-
-// your code ends here
+db.houses.deleteOne({
```

Comment:
This will remove the entire document rather than just the key.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```